brainiak/reprsimil/brsa.py

```diff
@@ -284,12 +270,12 @@ class BRSA(BaseEstimator, TransformerMixin):
         Note that nuisance regressor is not required from user. If it is
         not provided, DC components for each run will be included as nuisance
         regressor regardless of the auto_nuisance parameter.
-    n_nureg: int or string. Default: 'opt'
+    n_nureg: int or None. Default: None
```

Comment:
Do you mean, writing it as:
`n_nureg: Optional[int]`, period?
Does it also apply to other parameters whose defaults are `None`, such as `rank`, `random_state`, `space_smooth_range` and `inten_smooth_range`?